xcodeproj/repositories.bzl

```diff
@@ -134,7 +134,7 @@ swift_library(
 
     # Main branch as of 2022-08-15. Contains Hashable implementation for
     # XCScheme.BuildableReference
```

Comment:
Can you update the date and reason here as well (needing `XCScheme.ExecutionAction.shellToInvoke`)?